browser/ipfs/ipfs_tab_helper.h

```diff
@@ -59,6 +63,10 @@ class IPFSTabHelper : public content::WebContentsObserver,
                         const base::string16& body,
                         const GURL& link);
   GURL CreateAndCopyShareableLink(const ipfs::ImportedData& data);
+  void FileSelected(const base::FilePath& path,
```

Comment:
Make these two functions its own section (empty line separated) and add a comment for which class we're overriding for these two functions.